rogue_company_reverts.patch

```diff
@@ -40,7 +40,7 @@ index a50a988e7362cf3b20ea1c05e1f8eb9f6df2b996..6c8a9e80ba5f91f88e21478084e981f8
    tst-tls1-static \
    tst-tls1-static-non-pie \
    tst-tunables \
-+  tst-libc_dlvsym-static
++  tst-libc_dlvsym-static \
```

Comment:
That does not make sense, i guess.

Reply:
The backslash is needed because of how `make` interprets commands.  Without it, an error message is produced.  See #17.

Even looking at only the patch, the backslash can be seen to be missing because the new line is inserted in position where the existing line does have a backslash.

Reply:
I see, then its fine :+1: